src/vm/runners/builtin_runner/range_check.rs

```diff
@@ -121,30 +129,31 @@ impl RangeCheckBuiltinRunner {
         if vm.current_step < min_step {
             Err(MemoryError::InsufficientAllocatedCells)
         } else {
-            let builtin = BuiltinRunner::RangeCheck(self);
-            let used = builtin.get_used_cells(vm)?;
+            let used = self.get_used_cells(vm)?;
             let size = cells_per_instance
                 * safe_div(&bigint!(vm.current_step), &bigint!(ratio))
                     .map_err(|_| MemoryError::InsufficientAllocatedCells)?;
             Ok((used, size))
         }
     }
 
-    pub fn get_range_check_usage(&self, memory: &Memory) -> Option<(BigInt, BigInt)> {
-        let mut rc_bounds: Option<(BigInt, BigInt)> = None;
+    pub fn get_range_check_usage(&self, memory: &Memory) -> Option<(isize, isize)> {
```

Comment:
We could use usize here, the returned values will never be negative

Reply:
I presume they'll never overflow if casted directly into `isize`. Could you confirm that?